item.py

```diff
@@ -8,9 +9,18 @@ def __init__(self, id, title, datetime, status):
         self.datetime = datetime
 
     @classmethod
-    def from_raw_trello_card(cls, trello_card, status):
-        time = datetime.strptime(trello_card['dateLastActivity'],'%Y-%m-%dT%H:%M:%S.%fZ')
-        id = trello_card['id']
-        title = trello_card['name']
+    def from_raw_card(cls, card):
+        time = datetime.strptime(card['dateLastActivity'],'%Y-%m-%dT%H:%M:%S.%fZ')
+        id = card['_id']
+        title = card['name']
+        status = card['idList']
+
+
+        # if trello_card['idList'] == os.environ['todo_list_id']:
```

Comment:
Let's delete it rather than comment it out, we can recover it with git if needed